gen.go

```diff
@@ -29,6 +35,68 @@ func main() {
 		log.Fatal("USAGE: go run -tags generate gen.go <version>")
 	}
 	version := os.Args[1]
+	//generateProtoBinaries(version)
```

Comment:
should it be removed?

Reply:
no, that was just done to speed up local testing. Reverted it.